model/schema.json

```diff
@@ -442,6 +442,24 @@
           "description": "API key the Elastic Agent uses to authenticate with elasticsearch",
           "type": "string"
         },
+        "default_api_key_history": { 
+          "description": "Default API Key History",
+          "type": "array",
```

Comment:
Could we maybe set this to index: false instead? https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-index.html The potential problem I see with this structure if we index is that each entry creates a separate document in lucene which creates unnecessary overhead if we never have to query the data.

Reply:
yep. will have that set to false for elasticsearch PR for .fleet-agents mapping.